receipt/src/main/res/values/transaction_types.xml

```diff
@@ -0,0 +1,103 @@
+    <string name="bank_account_transfer_fee">Bank Account Return Fee</string>
+    <string name="bank_account_transfer_return">Bank Account Return</string>
+    <string name="bank_account_transfer_return_fee">Bank Account Return Fee</string>
+    <string name="transfer_to_bank_account">Bank Account</string>
+
+    <!--Card-specific types-->
+    <string name="card_activation_fee">Card Activation Fee</string>
+    <string name="card_activation_fee_waiver">Card Activation Fee Waiver</string>
+    <string name="card_fee">Card Fee</string>
+    <string name="manual_transfer_to_prepaid_card">Card Load</string>
+    <string name="prepaid_card_balance_inquiry_fee">Balance Inquiry Fee</string>
+    <string name="prepaid_card_cash_advance">Cash Advance</string>
+    <string name="prepaid_card_disputed_charge_refund">Disputed Charge Refund</string>
+    <string name="prepaid_card_dispute_deposit">Disputed Card Deposit</string>
+    <string name="prepaid_card_domestic_cash_withdrawal_fee">ATM Withdrawal</string>
+    <string name="PREPAID_CARD_EXCHANGE_RATE_DIFFERENCE"></string>
+    <string name="prepaid_card_manual_unload">Card Unload</string>
+    <string name="prepaid_card_overseas_cash_withdrawal_fee">ATM Withdrawal</string>
+    <string name="prepaid_card_pin_change_fee">PIN Change Fee</string>
+    <string name="prepaid_card_refund">Refund</string>
+    <string name="prepaid_card_replacement_fee">Card Replacement Fee</string>
+    <string name="prepaid_card_sale">Transaction</string>
+    <string name="prepaid_card_sale_reversal">Reversed Transaction</string>
+    <string name="prepaid_card_unload">Card Unload</string>
+    <string name="transfer_to_prepaid_card">Card Load</string>
+    <!---->
+    <!--Donation types-->
+    <string name="donation">Donation</string>
+    <string name="donation_fee">Donation Fee</string>
+    <string name="donation_return">Donation Return</string>
+    <!---->
+    <!--Merchant payment types-->
+    <string name="merchant_payment">Merchant Payment</string>
+    <string name="merchant_payment_fee">Merchant Payment Fee</string>
+    <string name="merchant_payment_refund">Merchant Payment Refund</string>
+    <string name="merchant_payment_return">Merchant Payment Return</string>
+    <!---->
+    <!--MoneyGram types-->
+    <string name="moneygram_transfer_return">MoneyGram Return</string>
+    <string name="transfer_to_moneygram">MoneyGram</string>
+    <!---->
+    <!--Paper check types-->
+    <string name="paper_check_fee">Paper Check Fee</string>
+    <string name="paper_check_refund">Paper Check Refund</string>
+    <string name="transfer_to_paper_check">Paper Check</string>
+    <!---->
+    <!--User and program account types-->
+    <string name="account_closure">Account Closure</string>
+    <string name="account_closure_fee">Account Closure Fee</string>
+    <string name="account_unload">Funds Unloaded</string>
+    <string name="dormant_user_fee">Domant User Account Fee</string>
```

Comment:
typo in Domant - it should be Dormant. Same issue in the line bellow

Reply:
renamed